db/item_db.conf

```diff
@@ -150872,6 +150872,27 @@ item_db: (
 
 //== New Etc Items =========================================
 {
+	Id: 22508
+	AegisName: "Para_Team_Mark_"
+	Name: "Eden Group Mark"
+	Type: 11
+	Buy: 0
+	Delay: 1200000
+	Trade: {
+		nodrop: true
+		notrade: true
+		noselltonpc: true
+		nocart: true
+		nostorage: true
+		nogstorage: true
+		nomail: true
+		noauction: true
+	}
+	Script: <"
+		unitskilluseid getcharid(3),"AL_TELEPORT",3;
```

Comment:
I didn't understand the coordinate problem. I didn't change any coordinate, could you explain it a little more? Thanks!